src/fragments/lib/graphqlapi/js/getting-started.mdx

```diff
@@ -88,23 +112,74 @@ await API.graphql(graphqlOperation(updateTodo, { input: { id: todoId, name: "Upd
 /* delete a todo */
 await API.graphql(graphqlOperation(deleteTodo, { input: { id: todoId }}));
 ```
+
+</Block>
+</BlockSwitcher>
+
 - __Queries__ - read data from the API (list, get operations)
 
+<BlockSwitcher>
+<Block name="TypeScript">
+
+```ts
+import { API, graphqlOperation } from 'aws-amplify';
+import { GraphQLQuery } from '@aws-amplify/api';
+import { listTodos } from './graphql/queries';
+import { ListTodosQuery } from './API';
+
+const todos = await API.graphql<GraphQLQuery<ListTodosQuery>>(graphqlOperation(listTodos));
+```
+
+</Block>
+
+
+<Block name="JavaScript">
+
 ```js
 import { API, graphqlOperation } from 'aws-amplify';
 import { listTodos } from './graphql/queries';
 
 const todos = await API.graphql(graphqlOperation(listTodos));
 ```
 
+</Block>
+</BlockSwitcher>
+
 - __Subscriptions__ - subscribe to changes in data for real-time functionality (onCreate, onUpdate, onDelete)
 
+<BlockSwitcher>
+<Block name="TypeScript">
+
+```ts
+import { API, graphqlOperation } from 'aws-amplify';
+import { GraphQLSubscription } from '@aws-amplify/api';
+import { onCreateTodo } from './graphql/subscriptions';
+import { OnCreateTodoSubscription } from './API';
+
+// Subscribe to creation of Todo
+const sub = API.graphql<GraphQLSubscription<OnCreateTodoSubscription>>(
+    graphqlOperation(onCreateTodo)
+).subscribe({
+    next: (todoData) => {
+      console.log(todoData);
+      // Do something with the data
+    }
```

Comment:
This is probably out of scope of this PR, but we may want to take another pass through all of these snippets and ensure we're demonstrating the shape/unpacking of these response objects. `todoData` is a little misleading here, when the actual "`Todo` data" is buried under `todoData.value.data?.onCreateTodo` (or something like that).

Reply:
I was debating whether to unwrap it or not. I thought the entire subscription payload may be of value to some customers, but perhaps at least a better param name here would clear things up